docs/physics/spectrum/basic.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>"You may notice that the bins do not appear to be uniformly spaced with respect to wavelength." --&gt; "You may notice that the bins are not uniformly spaced with respect to wavelength."</p><p> </p><p>Also, I would say "<strong>noise </strong>i.e. the effects of the random nature of Monte Carlo simulations" instead of "<strong>noise</strong>, or the effects of the random nature of Monte Carlo simulations".</p><p> </p><p>"To solve these problems," -- "To solve this problem,"</p><p> </p><p>I would maybe add at the end that TARDIS does all of this for you and that you can access the result via <code>sim.runner.spectrum.luminosity_density_nu</code> ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>